src/Core/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -1,10 +1,10 @@
 using System;
+using GraphQL;
 using GraphQL.Instrumentation;
-using GraphQL.Server.Internal;
-using Microsoft.Extensions.DependencyInjection;
+using GraphQL.Server;
 using Microsoft.Extensions.DependencyInjection.Extensions;
 
-namespace GraphQL.Server
+namespace Microsoft.Extensions.DependencyInjection
```

Comment:
This is namespace of `this` argument.

Reply:
Correct me if I'm wrong, but I was under the impression that if a namespace was imported, all extensions in static classes of that namespace are then available.  So with this change you would need to have `using Microsoft.Extensions.DependencyInjection;` in order to use these extensions.  Is that what is desired?  No opinion here, just want to clarify.

Reply:
Right. The fact is that in most cases the namespace of the `this` parameter has already been imported. Due to this, IntelliSense discoverability is provided. See https://github.com/graphql-dotnet/server/blob/develop/samples/Samples.Server/Startup.cs#L10

`using Microsoft.Extensions.DependencyInjection` is already here.

Reply:
It's actually possible in VS2019 >16.4 (and iirc also Rider) to do this without having them in the same namespace.
For VS2019:
![image](https://user-images.githubusercontent.com/8708965/75421816-66bfa680-593b-11ea-843a-c4da1d101a65.png)

Note that this is not a suggestion to change it back, simply wanted to make sure you had all the information for when making the decision.

Reply:
OK. Thanks for information, I did not know about this setting.

Reply:
Didn't know about this setting either. VS has always baffled me when it sometimes finds extensions and sometimes doesn't. I'm guilty of making my namespaces either non-existant or very simple as a result so people find them. Thanks for the tip!

Reply:
![image](https://user-images.githubusercontent.com/157336/75526728-9d1c2500-5a12-11ea-9ae3-00e8855a725f.png) LOL. I wonder what that means exactly...

Reply:
It means partially enabled. You can press `Alt+A` when the suggestion window is open. But I just have it fully enabled at all times, it's much easier and great for discovery.